Modify rule S2761: LaYC multiple unary operators (#2180)
SonarSource/rspec: rules/S2761/description.adoc

> Calling the ``++!++`` or ``++~++`` prefix operator twice does nothing: the second invocation undoes the first. Such repetitions are usually typos.

Reviewer: ```suggestion
The repetition of a prefix operator (``++!++``, or ``++~++``) is usually a typo. The second operator invalidates the first one.
```
